src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -64,6 +65,8 @@ public GenevaLogExporter(GenevaExporterOptions options)
             var tempTableMappings = new Dictionary<string, string>(options.TableNameMappings.Count, StringComparer.Ordinal);
             foreach (var kv in options.TableNameMappings)
             {
+                Guard.ThrowIfNull(kv.Value);
```

Comment:
I would recommend including the Key name in the exception message here.

Reply:
Would it be better to do this validation in the `GenevaExporterOptions` class, like how it's done for `PrepopulatedFields`: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs#L36-L73